### PR TITLE
KnockbackPlus module

### PR DESCRIPTION
**Pull request by Beta-51**
makes you give a little extra knockback when hitting people by sprinting and sending a packet.

### CODE REVIEW

File: src/main/java/coffee/client/feature/module/impl/combat/KnockbackPlus.java [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+import net.minecraft.util.math.Vec3d;
+
+import java.util.Objects;
+
+public class KnockbackPlus extends Module {
+
+
+    public KnockbackPlus() {
+        super("KnockbackPlus", "Makes you give the thing you're hitting extra knockback", ModuleType.COMBAT);
+        Events.registerEventHandler(EventType.PACKET_SEND, event1 -> {
```
Comment (0x3C50) [inline src/main/java/coffee/client/feature/module/impl/combat/KnockbackPlus.java]: Use the more modern event listener, this shit is legacy

Comment (Beta-51) [reply src/main/java/coffee/client/feature/module/impl/combat/KnockbackPlus.java]: i'm still very very new to this stuff, but i'll try using that and get back to this

Comment (Beta-51) [reply src/main/java/coffee/client/feature/module/impl/combat/KnockbackPlus.java]: yeah i'm gonna be real i dunno what you're talking about

Comment (0x3C50) [reply src/main/java/coffee/client/feature/module/impl/combat/KnockbackPlus.java]: the `@EventListener` annotation shit, look in any other module on how to use it

Comment (Beta-51) [reply src/main/java/coffee/client/feature/module/impl/combat/KnockbackPlus.java]: alright, did that, should be good now

**Pull request merged**